QueryBuilder/Clauses/Condition.cs

```diff
@@ -74,7 +74,7 @@ internal class WhereIsOfModelCondition : SimpleCondition
 
         public override string ToString()
         {
-            return $"{IsOfModel}({Alias}, '{GetModelWithVersion(Model, minTwinVersion)}')";
+            return string.IsNullOrEmpty(Alias) ? $"{IsOfModel}('{GetModelWithVersion(Model, minTwinVersion)}')" : $"{IsOfModel}({Alias}, '{GetModelWithVersion(Model, minTwinVersion)}')";
```

Comment:
Alias shouldn't be defined here. The conditions should assume Alias is always passed. Maybe we should add a constructor to SimpleCondition that accepts an alias parameter to enforce the right usage. Please add that

Reply:
When using `WhereIsOfModel()` made sense to manage a lack of alias here. We want to either get the one provided by the user/use the one from the list or show none if the user intentionally doesn't wish to an alias.